packages/components/src/components/atoms/UnoCardFace/UnoCardFace.tsx

```diff
@@ -0,0 +1,687 @@
+      className="st2"
+      clipRule="evenodd"
+    />
+    <path
+      fillRule="evenodd"
+      d="M61.567 91.602l.03.059-.95.356c-.03 0-.06.03-.06.03-.03 0-.06.03-.089.03l-.95.356-.03-.06c-4.54 1.188-7.657-1.187-9.29-5.55l-3.562-9.558 4.542-1.692 3.65 9.766c.476 1.246 1.336 2.345 2.79 2.404.416.03.862-.06 1.247-.208a2.988 2.988 0 001.099-.683c1.038-.979.98-2.374.504-3.62l-3.68-9.766 4.541-1.692 3.621 9.676c1.633 4.393.831 8.104-3.413 10.152z"
+      className="st2"
+      clipRule="evenodd"
+    />
+    <path
+      fill="#fff"
+      fillRule="evenodd"
+      d="M60.795 91.305l-.386.148c-.03 0-.06.03-.089.03-.03 0-.06.03-.089.03l-.386.148c-4.511 1.425-7.51-.742-9.083-5.016l-3.354-8.994 3.414-1.276 3.443 9.201c.564 1.484 1.632 2.73 3.324 2.79.505.03 1.01-.06 1.484-.237.475-.178.95-.445 1.306-.802 1.217-1.157 1.217-2.79.653-4.274l-3.443-9.201 3.414-1.277 3.413 9.113c1.603 4.274.742 7.747-3.621 9.617z"
+      className="st3"
+      clipRule="evenodd"
+    />
+    <path
+      fill="#f8da27"
+      fillRule="evenodd"
+      d="M47.943 77.889l2.641-.98 3.295 8.816c1.098 2.908 3.414 3.502 5.343 2.79 1.93-.742 3.324-2.701 2.196-5.61l-3.294-8.816 2.641-.98 3.265 8.727c1.306 3.473 1.306 7.272-3.77 9.261-.029 0-.059.03-.088.03-.03 0-.06.03-.09.03-4.986 1.78-7.628-1.07-8.963-4.631z"
+      className="st4"
+      clipRule="evenodd"
+    />
+    <path
+      fill="#58585a"
+      fillRule="evenodd"
+      d="M70.323 70.142l1.632-1.663.92 2.464-1.632 1.662zm1.217 3.294l1.632-1.662.357.92-1.633 1.662zm-16.117 2.286l1.632-1.662.92 2.463-1.632 1.662zm1.217 3.294l1.632-1.662.356.92-1.632 1.663zm-9.35 6.056l1.632-1.663.92 2.464-1.632 1.662zm1.246 3.294l1.633-1.662.06.119c.118.267.207.534.326.801l-1.662 1.662zm39.21-18.729l1.663-1.662c.593.475 1.098 1.157 1.454 1.929l-1.633 1.633c-.356-.713-.86-1.395-1.484-1.9zm1.84 2.701l1.603-1.633c.12.357.179.683.238 1.04l-1.573 1.572v-.03a6.61 6.61 0 00-.267-.949z"
+      className="st5"
+      clipRule="evenodd"
+    />
+  </g>
+);
+
+export const UnoCardFace: React.FC<UnoCardFaceProps> = ({ card, className }) => {
+  const classes = useStyles();
+
+  const isBackFace = !card;
+  const fill = isBackFace ? '' : colors[card.color];
+
+  const renders: { [key in UnoCardValue]: () => JSX.Element } = {
+    0: () => renderN0(fill),
+    1: () => renderN1(fill),
+    2: () => renderN2(fill),
+    3: () => renderN3(fill),
+    4: () => renderN4(fill),
+    5: () => renderN5(fill),
+    6: () => renderN6(fill),
+    7: () => renderN7(fill),
+    8: () => renderN8(fill),
+    9: () => renderN9(fill),
+    jump: () => renderJump(fill),
+    reverse: () => renderReverse(fill),
+    plus2: () => renderPlus2(fill),
+    plus4: () => renderPlus4(),
+    joker: () => renderJoker(),
+  };
```

Comment:
I don't really like this logic here, I'd rather have a dumb svg component like we have for playing cards. One reason is that we have it as a component because we couldn't figure out how to use and export directly an svg file, and if we want to try this again in the future we will be constrained by this logic.

Reply:
I think we should discuss about this topic - through here it won't be super effective.

I am not super fan of having three components to have the PlayingCard faces. I don't think the logic is complex enough to justify the split into three components.
Furthermore, I find a pitty that the SVG component from PlayingCard is obliged to render all possible `def`s to only then render a `use` tag that will reference one of them. The way I propose, I only render what's going to be used.

Reply:
Yeah I think I both agree and disagree with you 😄 
I think it would be better to discuss it in a call for sure, so let's try to have one this week 🙂

Reply:
As discussed, we will move the internal functions to sub-components